docs/app/Examples/elements/Icon/IconSet/IconSetExamples.js

```diff
@@ -2,7 +2,26 @@ import _ from 'lodash'
 import React, { Component } from 'react'
 import ComponentExample from 'docs/app/Components/ComponentDoc/ComponentExample'
 import ExampleSection from 'docs/app/Components/ComponentDoc/ExampleSection'
-import { iconCats as categories } from 'src/utils/semanticUtils'
+import { iconCategories as categories } from 'src/utils/semanticUtils'
+
+const template = (content) => (`
+  import React, { Component } from \'react\'
+  import { Icon } from \'stardust\'
+  
+  export default class IconCategoryExample extends Component
```

Comment:
Awesome job pulling that off 😉  Could we insert category name into the class name and put the first import statement on the first line?

```js
const template = (content) => (`import React, { Component } from \'react\'
  import { Icon } from \'stardust\'

  export default class Icon${icon.category}Example extends Component {
    ...
  }
`
```